src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.xaml

```diff
@@ -3,8 +3,8 @@
 					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 					xmlns:utu="using:Uno.Toolkit.UI"
-					mc:Ignorable="d">
-
+					xmlns:void="used for quickly commenting out node or attribute"
```

Comment:
@Xiaoy312, @kazo0 are we really keeping void namespace like this?

Reply:
it is a feature of xaml so why not, plus this will be hidden from the consumer